src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -596,16 +578,6 @@ internal static bool GlobalMemoryStatusEx(ref MEMORYSTATUSEX buffer)
         internal const int FIND_FROMSTART = 0x00400000; // look for value in source, starting at the beginning
         internal const int FIND_FROMEND = 0x00800000; // look for value in source, starting at the end
```

Comment:
done. Note that ILLink will never tell us about these, as it shows all consts as "removed". It would be nice to make it more useful for dead code removal eg to not show default empty ctors, inlined constants,. .